lambda-code/reliability/lib/dataLayer.ts

```diff
@@ -124,6 +124,7 @@ export async function saveToVault(
             CreatedAt: Number(createdAt),
             SecurityAttribute: securityAttribute,
             Status: "New",
+            "Status#CreatedAt": `NEW#${Number(createdAt)}`,
```

Comment:
Is the all capital letters a constraint from DynamoDB? Or maybe a best practice? If not, I wonder if we should not just use the same format as the one we use for the `Status` field.

```suggestion
            "Status#CreatedAt": `New#${Number(createdAt)}`,
```

Reply:
I thought I’d be following the pattern I saw for NAME_OR_CONF, but I also like the idea of sticking with what we used for Status. Perhaps somewhere else in our codebase, we could use `${status}#${Number(createdAt)}`.